src/components/MyThreads/ThreadItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Separator } from "@/components/ui/separator";
+
+export interface Props {
+  type: "post" | "comment";
+  channel: "응원" | "칭찬" | "무능";
+  time?: string;
+  content?: string;
+}
+
+const MyThreadItem = ({ type, channel, time, content }: Props) => {
+  const element: {
+    [type: string]: JSX.Element;
+  } = {
+    post: (
+      <div className="mb-2 text-lg font-normal">
+        안녕하세요 제가 쓴 글입니다 프롱이들 다 착해요.
+      </div>
+    ),
+    comment: (
+      <>
+        <p className="text-sm text-muted-foreground">다음에 댓글 남김: {content}</p>
+        <div className="mb-2 text-lg font-normal">ㅋㅋ저도요</div>
+      </>
+    ),
+  };
+  return (
+    <>
+      <div className="flex items-center justify-between gap-6 pt-3">
+        <p className="text-sm text-muted-foreground">#{channel}게시판</p>
+        <p className="text-xs font-extralight">{time ?? "오후 12:33"}</p>
+      </div>
+      {element[type]}
+      <Separator />
+    </>
```

Comment:
이름이`Item`이듯이 이 친구는 list 요소인 것을 나타내는 것 같아요! 그래서 li태그를 사용하는게 좋을 것 같습니다.
```suggestion
    <li>
      <div className="flex items-center justify-between gap-6 pt-3">
        <p className="text-sm text-muted-foreground">#{channel}게시판</p>
        <p className="text-xs font-extralight">{time ?? "오후 12:33"}</p>
      </div>
      {element[type]}
      <Separator />
    <li/>
```
li는 반드시 ul 태그를 부모로 가져야 하니 `ThreadBody` 부분도 아래와 같이 바꿔야 좋을 것 같아요!
```js
<ul>
  <ThreadItem />
</ul>
```